styletesla.css

```diff
@@ -1,97 +1,640 @@
+	width: 22px;
+	height: 22px;
+	margin-right: 5px;
+}
+.know { 
+	font: 13px/16px montserratbold;
+	cursor: pointer;
+}
+.button__order {
+    border: 2px solid white;
+	color: white;
+	background: #E51937;
+	font: 13px/16px montserratbold;
+	text-transform: uppercase;
+	margin-right: 30px;
+}
+/*планшет*/
+@media (max-width: 1270px) {
+	.sidebar {
+	width: 35%;
+	}
+}
+@media (max-width: 1165px) {
+.advantage {
+	height: 900px;
+	width: 100%;
+	background: url('img/rulj-planshet.png') top no-repeat; 
+	background-size: contain;
+	display: flex;
+	flex-direction: column;
 }
+.advantage__title {
+	padding-left: 54%;
+	margin-top: 3%;
+	}
+.sidebar {
+	width: 100%;
+	height: 51%;
+	padding-left: 0;
+	display: grid;
+	grid-template-columns: 53% 47%;
+	grid-template-rows: 80% 20%;
+    }
+.sidebar__title, .sidebar__discription {
+	grid-column: 1/2;
+	grid-row: 1/2;
+	padding-left: 50px;
+    }
+.sidebar__discription {
+	padding-top: 90px;
+}
+.list{
+	grid-column: 2/3;
+	padding-top: 35px;
+	margin-bottom: 0;
+}
+.button {
+	grid-column: 1/3;
+	display: flex;
+	justify-content: center;
+   }
+.button__more {
+	padding-right: 120px;
+}
+}
+@media (max-width: 790px) {
+.sidebar {
+	width: 100%;
+	height: 55%;
+    }
 .sidebar__title, .sidebar__discription {
-	color:white;
+	padding-left: 25px;
+	}
+.list {
+	padding-left: 40px;
+	}
 }
+@media (max-width: 670px) {
+.advantage__title, .list, .button {
+		display: none;
+	}
+.advantage {
+	height: 580px;
+	width: 100%;
+	/*можно почистить?*/
+	background: url('img/rulj-planshet.png') top no-repeat; 
+	background-size: contain;
+	display: flex;
+	flex-direction: row;
+	align-items: flex-end;
+}
+.sidebar {
+	display: flex;
+	flex-direction: column;
+	height: 58%;
+	}
+.sidebar__title, .sidebar__discription {
+	margin: 0 auto;
+	}
 .sidebar__title {
-	padding-top: 51px;
-	margin: 0;
+	padding: 20px 0;
+}
+.sidebar__discription {
+	padding: 0;
+}	
 }
+@media (max-width: 383px) {
+.advantage {
+	height: 568px;
+	}
+.sidebar {
+	height: 64%;
+	}
+}
+
+/*3section*/
+.safety {
+	/*height: 100vh;*/
+	width: 100%;
+	display: grid;
+	grid-template-columns: 20% 80%;
+	grid-template-rows: 100%;
+}
+.sidebar {
+    grid-column: 1/3;
+    grid-row: 1/2;
+}
+.protect__list {
+	margin-bottom: 100px ;
+}
+.protect__list_link {
+	display: flex;
+	justify-content: space-between;
+    margin-right: 90px;
+    margin-bottom: 16px;
+	font: 17px/21px montserratbold;
+}
+.list_star {
+	height: 22px;
+}
+.security {
+	display: flex;
+	align-items: center;
+}
+.security__icon {
+	width: 62px;
+	height: 74px;
+	margin-right: 20px;
+}
+.star {
+	font: 33px/36px montserratsemibold;
+}
+.star_security {
+	font: 25px/30px montserratsemibold ;
+}
+.safety__img {
+	grid-column: 2/3;
+	grid-row: 1/2;
+	background: url('img/ModelS.png') right no-repeat;
+	background-position-y: 17%;
+	background-size: 70%;
+}
+.carinside {
+	padding-top: 175px;
+	padding-left: 50px;
+	width: 90%; 
+}
+.footer {
+	grid-column: 2/3;
+	display: flex;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-12.04/pullRequest?prid=5512124)